### PR TITLE
get: add get sub-command dr-health

### PR DESCRIPTION
**Pull request by subhamkrai**
adding get sub-command `dr-health` which prints the
ceph status of peer cluster.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ The ODF CLI tool provides configuration and troubleshooting commands for OpenShi
 - `odf get`:
   - `recovery-profile`: Get the recovery profile value.
   - `health`: Check health of the cluster and common configuration issues.
+  - `dr-health [ceph status args]`: Print the ceph status of a peer cluster in a mirroring-enabled environment thereby validating connectivity between ceph clusters. Ceph status args can be optionally passed, such as to change the log level: --debug-ms 1.
```
Comment (travisn) [inline README.md:13]: The "ceph status args" are specifically passed to the `ceph status` command and nothing else?

File: cmd/odf/get/dr_health.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	"github.com/rook/kubectl-rook-ceph/pkg/dr"
+	"github.com/spf13/cobra"
+)
+
+var drHealthCmd = &cobra.Command{
+	Use:                "dr-health",
+	Short:              "Print the ceph status of a peer cluster in a mirroring-enabled cluster.",
+	DisableFlagParsing: true,
+	Args:               cobra.MaximumNArgs(2),
+	Example:            "rook-ceph dr health [Arg]",
```
Comment (travisn) [inline cmd/odf/get/dr_health.go:14]:
```suggestion
	Example:            "rook-ceph dr health [ceph status args]",
```

File: README.md (hunk 1 above)
Comment (subhamkrai) [reply README.md:13]: yes, correct

File: cmd/odf/get/dr_health.go (hunk 2 above)
Comment (subhamkrai) [reply cmd/odf/get/dr_health.go]: done

**Pull request merged**